Aio.Multidataset.Rest.Connector/StreamingTelemetryConnectorWorker.cs

```diff
@@ -1,25 +1,24 @@
-// Copyright(c) Microsoft Corporation.
```

Comment:
We may not need to class, we could potentially just register the base class directly in DI.